### PR TITLE
Nojira/user identity sandbox env

### PR DESCRIPTION
**Pull request by yenobi**

### CODE REVIEW

**Review by Farzannajipour — APPROVED** [review]

File: src/environments/environment.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const environment = {
 
 export const authConfig: AuthConfig = {
   // Url of the Identity Provider
-  issuer: 'https://identity.ret-us-l.rnd.live.backbaseservices.com/auth/realms/backbase',
+  issuer: 'https://identity.dev.sdbx.live.backbaseservices.com/auth/realms/retail',
```
Comment (georgejolema) [inline src/environments/environment.ts:33]: can you also check this https://github.com/Backbase/golden-sample-app/blob/master/docker-compose.yml#L16 and update if possible

File: src/environments/type.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Provider } from "@angular/core";
+
+export interface Environment {
+	production: boolean,
+	apiRoot: string,
+	mockProviders: Provider[],
+	locales: string[],
+	apiURL?: string,
```
Comment (santam85) [inline src/environments/type.ts:8]: what's the difference between apiRoot and apiURL?

**Review by santam85 — CHANGES_REQUESTED** [review]

**Review by georgejolema — APPROVED** [review]

File: src/environments/type.ts (hunk 2 above)
Comment (georgejolema) [reply src/environments/type.ts:8]: it should only be apiRoot now

Comment (yenobi) [reply src/environments/type.ts]: removed `apiURL` from type and the actual value to avoid confusion

**Review by santam85 — APPROVED** [review]

**Pull request merged**